content/en/account_management/billing/aws.md

```diff
@@ -9,6 +9,9 @@ aliases:
 
 Datadog bills for AWS hosts running the Datadog Agent and all EC2 instances picked up by the Datadog-AWS integration. **You are not billed twice** if you are running the Agent on an EC2 instance picked up by the AWS integration.
 
+**IMPORTANT**: Datadog uses EC2 instance metadata to ensure you aren't billed twice for hosts both running the agent and being picked up by the AWS integration. If your EC2 intances are configured to require the use of [Instance Metadata Service Version 2 (IMDSv2)][6], then you must set the parameter `ec2_prefer_imdsv2` to `true` in your [Agent configuration][7] to avoid double billing.
```

Comment:
```suggestion
**IMPORTANT**: Datadog uses EC2 instance metadata to ensure you aren't billed twice for hosts both running the agent and being crawled by the AWS integration. If your EC2 instances are configured to require the use of [Instance Metadata Service Version 2 (IMDSv2)][6], then you must set the parameter `ec2_prefer_imdsv2` to `true` in your [Agent configuration][7] to avoid double-billing.
```